docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -492,6 +495,41 @@ To configure settings globally for all Ingress rules, the `limit-rate-after` and
 
 The client IP address will be set based on the use of [PROXY protocol](./configmap.md#use-proxy-protocol) or from the `X-Forwarded-For` header value when [use-forwarded-headers](./configmap.md#use-forwarded-headers) is enabled.
 
+### Global Rate Limiting
```

Comment:
Please add a warning saying this feature is not compatible with the existing rate limits?

Reply:
It is actually compatible and in certain cases desirable to be used together. One can use local throttling to protect global throttling backend from large traffic spikes. This is also noted at https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_features/global_rate_limiting

> Note that Envoy also supports local rate limiting. Local rate limiting can be used in conjunction with global rate limiting to reduce load on the global rate limit service. For example, a local token bucket rate limit can absorb very large bursts in load that might otherwise overwhelm a global rate limit service. Thus, the rate limit is applied in two stages. The initial coarse grained limiting is performed by the token bucket limit before a fine grained global limit finishes the job.

Reply:
> It is actually compatible and in certain cases desirable to be used together. 

Yes, but the behavior is different. The existing rate limit applies per pod.

Reply:
added a note